### PR TITLE
Correct MySQL Shell/Client section

### PR DESCRIPTION
**Pull request by Oreoxmt**
Signed-off-by: Aolin <aolin.zhang@pingcap.com>

### What is changed, added or deleted? (Required)

There are two official command-line clients for MySQL, both open-source and released by the Oracle MySQL team.

1. MySQL Client (`mysql`), which is part of the MySQL Server code base, but can be installed separately via `yum install mysql` on most YUM based linux distributions. With [the offical YUM repo](https://dev.mysql.com/doc/refman/8.0/en/linux-installation-yum-repo.html) this becomes `yum install mysql-community-client`. This client is written in C and has existed for as long as MySQL does.

2. MySQL Shell (`mysqlsh`), which is a separate code base and can be installed separately. This is a newer tool and supports SQL, but also JS and Python. It has a more modern code base and is more extendable than the 'old' client.

It looks like these two were mixed up into a single section of the docs.

Simpletab preview:

image

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/8779
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-connect-to-tidb.md [hunk 1]
```diff
@@ -10,15 +10,29 @@ aliases: ['/zh/tidb/dev/connect-to-tidb']
 
 TiDB 支持 [MySQL 客户端/服务器协议](https://dev.mysql.com/doc/internals/en/client-server-protocol.html)。这使得大多数客户端驱动程序和 ORM 框架可以像连接到 MySQL 一样地连接到 TiDB。
 
+你可以选择使用 MySQL Client 或 MySQL Shell 连接到 TiDB。
+
 ## MySQL Client
 
-你可以使用 MySQL Client 作为 TiDB 的命令行工具。在 [MySQL Shell 官方文档](https://dev.mysql.com/doc/mysql-shell/8.0/en/mysql-shell-install.html) 你可以找到不同操作系统的安装方式。在安装完后你可以使用如下命令行连接到 TiDB：
+你可以使用 MySQL Client 作为 TiDB 的命令行工具连接到 TiDB。下面以基于 YUM 的 Linux 发行版为例，介绍如何按照 MySQL Client。
```
Comment (Icemap) [inline develop/dev-guide-connect-to-tidb.md:17]: Did we need to add the `<SimpleTabs>` tag to give some other platform examples to the audience?

**Review by Icemap — APPROVED** [review]
Rest LGTM

File: develop/dev-guide-connect-to-tidb.md (hunk 1 above)
Comment (Oreoxmt) [reply develop/dev-guide-connect-to-tidb.md:21]: Done https://github.com/pingcap/docs-cn/pull/12987/commits/ac60843553b4a5c773884ef8d14b4ec7c27913ed

Preview:

image

File: develop/dev-guide-connect-to-tidb.md [hunk 2]
```diff
@@ -10,15 +10,39 @@ aliases: ['/zh/tidb/dev/connect-to-tidb']
...
+## MySQL
 
-你可以使用 MySQL Client 作为 TiDB 的命令行工具。在 [MySQL Shell 官方文档](https://dev.mysql.com/doc/mysql-shell/8.0/en/mysql-shell-install.html) 你可以找到不同操作系统的安装方式。在安装完后你可以使用如下命令行连接到 TiDB：
+你可以选择使用 MySQL Client 或 MySQL Shell 连接到 TiDB。
+
+<SimpleTab>
+
+<div label="MySQL Client">
+
+你可以使用 MySQL Client 作为 TiDB 的命令行工具连接到 TiDB。下面以基于 YUM 的 Linux 发行版为例，介绍如何按照 MySQL Client。
```
Comment (shichun-0415) [inline develop/dev-guide-connect-to-tidb.md:21]:
```suggestion
你可以使用 MySQL Client 作为 TiDB 的命令行工具连接到 TiDB。下面以基于 YUM 的 Linux 发行版为例，介绍如何安装 MySQL Client。
```

File: develop/dev-guide-connect-to-tidb.md [hunk 3]
```diff
@@ -10,15 +10,39 @@ aliases: ['/zh/tidb/dev/connect-to-tidb']
...
 ```shell
 mysql --host <tidb_server_host> --port 4000 -u root -p --comments
 ```
 
-注意：MySQL 命令行客户端在 5.7.7 版本之前默认清除了 [Optimizer Hints](/optimizer-hints.md#optimizer-hints)。如果需要在这些早期版本的客户端中使用 Hint 语法，需要在启动客户端时加上 `--comments` 选项。
+</div>
+
+<div label="MySQL Shell">
+
+你可以使用 MySQL Shell 作为 TiDB 的命令行工具连接到 TiDB。你可以参考 [MySQL Shell 文档](https://dev.mysql.com/doc/mysql-shell/8.0/en/mysql-shell-install.html)进行安装。安装完成后，你可以使用如下命令连接到 TiDB：
```
Comment (shichun-0415) [inline develop/dev-guide-connect-to-tidb.md:37]:
```suggestion
你可以使用 MySQL Shell 作为 TiDB 的命令行工具连接到 TiDB。参考 [MySQL Shell 文档](https://dev.mysql.com/doc/mysql-shell/8.0/en/mysql-shell-install.html)进行安装。安装完成后，你可以使用如下命令连接到 TiDB：
```

Comment (shichun-0415) [reply develop/dev-guide-connect-to-tidb.md:37]: 两个“你可以”挨着，只用一个会好一点。

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

**Pull request merged**